dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
 
-git+https://github.com/andrewsanchez/pytest-anki.git@anki-2.1.54-qt5
\ No newline at end of file
+git+https://github.com/andrewsanchez/pytest-anki.git@anki-2.1.54-qt5
```

Comment:
Should we use the same one we are using in the ankihub add-on repo?  Pedro changed it to https://github.com/ankipalace/pytest-anki.git

Reply:
Yes, I changed that now.